### PR TITLE
[DOCS] add comment to ALLOWED_METHODS section about enabling HTTP POST

### PR DESCRIPTION
**Pull request by spwoodcock**
## Issue

- The `ALLOWED_METHODS` variable is a bit hidden in the settings, under `Miscellaneous` https://docs.bunkerweb.io/latest/settings/#miscellaneous
- The default of `GET|POST|HEAD` is understandably very strict, as many users may run bunkerweb in front of simple applications that only serve content.
- Application developers using bunkerweb in front of their apps will most likely need `POST` (amonst others like `PATCH`, `DELETE`, etc).
- A HTTP `POST` require actually requires two requests from browsers:
  - A CORS pre-flight request using `OPTIONS` first.
  - Then the `POST` request if CORS doesn't fail.

## Solution

- Added a small note in the docs `ALLOWED_METHODS`, located: https://docs.bunkerweb.io/latest/security-tuning/#allowed-methods:

```
Note that if `POST` is required, then `OPTIONS` should also be specified to allow for the CORS pre-flight request.
```

## Notes

- It's probably possible to enable `OPTIONS` by default if `POST` is included, but this is probably a bad idea.
- Options should be defined explicitly rather than implicitly.
- Feel free to close if you feel this requirement is obvious. I thought it might help prevent silly mistakes (like I made during first configuration!).
- Also, a separate issue, but I'm on a pretty bad internet connection these days & it took me forever to clone the >100mb `main` branch! I wonder if this large size is necessary for something, or perhaps the git history could be cleaned up a bit.

### CODE REVIEW

**Comment by TheophileDiot** [comment]
Hi, thank you for opening this P/R. Indeed this detail is missing in the documentation and can be confusing.

File: docs/security-tuning.md
Comment (TheophileDiot) [inline docs/security-tuning.md:1]: Using the MkDocs admonitions feature could enhance the visibility of the warning. You can find more details here: [MkDocs Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types).

**Review by TheophileDiot — CHANGES_REQUESTED** [review]
Waiting for the recommended changes to be implemented.

**Comment by fl0ppy-d1sk** [comment]
Thanks @spwoodcock !

**Pull request merged**